Tools/clinic/clinic.py

```diff
@@ -1374,7 +1374,11 @@ def group_to_variable_name(group: int) -> str:
         adjective = "left_" if group < 0 else "right_"
         return "group_" + adjective + str(abs(group))
 
-    def render_option_group_parsing(self, f, template_dict):
+    def render_option_group_parsing(
+            self,
+            f: Function,
+            template_dict: TemplateDict
```

Comment:
FTR, `template_dict` is not used in `render_option_group_parsing`. As far as I understand, there's no official way of annotating a param as unused.